pkg/arvo/app/file-server.hoon

```diff
@@ -113,14 +122,17 @@
       ?:  (~(has by serving) url-base)
         ~|("url already bound to {<(~(got by serving) url-base.act)>}" !!)
       :-  [%pass url-base %arvo %e %connect [~ url-base] %file-server]~
-      this(serving (~(put by serving) url-base clay+clay-base.act public.act))
+      %_  this
+          serving
+        (~(put by serving) url-base clay+clay-base.act public.act spa.act)
+      ==
     ::
         %serve-glob
       =*  url-base  url-base.act
       ?:  (~(has by serving) url-base)
         ~|("url already bound to {<(~(got by serving) url-base.act)>}" !!)
       :-  [%pass url-base %arvo %e %connect [~ url-base] %file-server]~
-      this(serving (~(put by serving) url-base glob+glob.act public.act))
+      this(serving (~(put by serving) url-base glob+glob.act public.act %.y))
```

Comment:
So we pass `%.y` as the spa flag on `:glob`, which won't have the file — it mounts its state as `bundle.[hash].js`. Given the `?&` for spa and is-file flags, will this play nice?

Reply:
It works fine so long as the file name is lowercase. This is intended behavior given constraints around clay paths.

Reply:
I just mean because glob doesn't put it in clay, it pokes file-server to mount directly from state AFAIK.

Reply:
Not a concern given that is-file overrides that flag.